diff -uNr hsc/source/Makefile hsc-datadir/source/Makefile
--- hsc/source/Makefile	Mon Sep 14 17:35:42 1998
+++ hsc-datadir/source/Makefile	Sun Dec 15 14:08:42 2002
@@ -172,7 +172,7 @@
 	$(INSTALL) -m 755 hsc/hsc $(INSTDIR)/bin/hsc
 	$(INSTALL) -m 755 hsctools/hscdepp $(INSTDIR)/bin/hscdepp
 	$(INSTALL) -m 755 hsctools/hscpitt $(INSTDIR)/bin/hscpitt
-	$(INSTALL) -m 644 ../hsc.prefs $(INSTDIR)/lib/hsc.prefs
+	$(INSTALL) -m 644 ../hsc-html-40.prefs $(INSTDIR)/share/hsc.prefs
 
 
 #--------------------------------------------------------------------
diff -uNr hsc/source/hsclib/hscprc.h hsc-datadir/source/hsclib/hscprc.h
--- hsc/source/hsclib/hscprc.h	Mon Sep 21 11:50:34 1998
+++ hsc-datadir/source/hsclib/hscprc.h	Sun Dec 15 14:07:47 2002
@@ -50,7 +50,7 @@
 
 #elif (defined NEXTSTEP) || (defined UNIX) || (defined BEOS)
 #define CONFIG_FILE "hsc.prefs"
-#define CONFIG_PATH "/usr/local/lib/", "/usr/lib/"
+#define CONFIG_PATH "/usr/local/share/", "/usr/share/"
 #define OPTION_FILE "hsc.options"
 
 /* [3] */
